### PR TITLE
SR-14796: Don't lose '-sdk /' when dropping the trailing slash

### PR DESCRIPTION
**Pull request by buttaface**
This is a regression from [the C++ Driver](https://github.com/apple/swift/blob/35735e7d0c57b57953623eb069d3812cd95d25e6/lib/Driver/Driver.cpp#L1776), ~will add a test later today and this should be~ ready to go.

### CODE REVIEW

File: Sources/SwiftDriver/Driver/Driver.swift [hunk 1]
```diff
@@ -2100,7 +2100,7 @@ extension Driver {
     }
 
     // Delete trailing /.
-    sdkPath = sdkPath.map { $0.last == "/" ? String($0.dropLast()) : $0 }
+    sdkPath = sdkPath.map { $0.length > 1 && $0.last == "/" ? String($0.dropLast()) : $0 }
```
Comment (cltnschlosser) [inline Sources/SwiftDriver/Driver/Driver.swift:2103]: What is `length`? Should be `count` for String.

Comment (buttaface) [reply Sources/SwiftDriver/Driver/Driver.swift:2103]: `length` appears to work the same, but I will switch it to the more Swifty `count`.

**Comment by buttaface** [comment]
Added a test, this is ready for review and CI.

**Comment by artemcm** [comment]
@swift-ci please test

**Review by drexin — APPROVED** [review]
Thank you!

**Comment by buttaface** [comment]
@drexin, should I submit for the 5.5 branch too, so this doesn't regress when that is released, or is it not worth it? I'm not sure.

**Comment by drexin** [comment]
@buttaface Yeah, I think that would be good.

**Pull request merged**